docs/studios/unity/unity-sdk/creating-bots/javascript/validation-bots.mdx

```diff
@@ -149,6 +149,29 @@ export async function startScenario(rg) {
 
 These methods are in addition to the APIs defined in the [PlayTest Bots section](./playtest-bots#variables).
 
+### `recordValidation(validationName: string, result: "PASS" | "FAIL" | "WARNING", options);`
+
+Records a validation result that can later be seen in the Validations Dashboard. 
+
+#### Arguments
+Argument Name | Type | Default | Description
+---|---|---|---|
+`validationName` | `string` | | The name or message of the test to show in the UI
+`result` | `"PASS"` \| `"FAIL"` \| `"WARNING"` | | The result of the test
+`options` | `object` | `{}` | Additional options for the validation
+`options.icon` | `string` | | The icon to show in the UI. Can be `clock`, `heart`, `key`, or `sparkles`
```

Comment:
Why are these strings defined differently than the "result" type? (also reword of description)

```suggestion
`options.icon` | `"clock"` \| `"heart"` \| `"key"` \| `"sparkles"` | | The icon to show next to this validation in the Validations Dashboard
```